src/draw_parameters/mod.rs

```diff
@@ -79,14 +79,42 @@ pub use self::query::{AnySamplesPassedQuery, TransformFeedbackPrimitivesWrittenQ
 
 mod query;
 
-/// Function that the GPU will use for blending.
+/// Blend effect that the GPU will use for blending.
 ///
 /// Blending happens at the end of the rendering process, when the GPU wants to write the
 /// pixels over pixels that already exist in the framebuffer. The blending function allows
 /// you to choose how it should merge the two.
 ///
-/// If you want to add transparent objects one over another, the usual value
-/// is `Addition { source: SourceAlpha, destination: OneMinusSourceAlpha }`.
+/// If you want to add transparent objects one over another, use
+/// `Blend::alpha_blending()`.
+#[derive(Copy, Clone, Debug, PartialEq)]
+pub struct Blend {
+    /// The blending function for color channels.
+    pub color: BlendingFunction,
+    /// The blending function for alpha channels.
+    pub alpha: BlendingFunction,
+    /// A constant color that can be used in the blending functions.
+    pub constant_value: (f32, f32, f32, f32),
+}
+
+impl Blend {
+    /// Returns a blend effect to add transparent objects over others.
+    pub fn alpha_blending() -> Blend {
+        Blend {
+            color: BlendingFunction::Addition {
+                source: LinearBlendingFactor::SourceAlpha,
+                destination: LinearBlendingFactor::OneMinusSourceAlpha,
+            },
+            alpha: BlendingFunction::Addition {
+                source: LinearBlendingFactor::One,
+                destination: LinearBlendingFactor::One
```

Comment:
I think the values here should be `SourceAlpha` and `OneMinusSourceAlpha` as well. I'm not sure but I think that if you use HDR you could end up with an alpha values > 1.0.